inference-engine/tests/functional/inference_engine/ngraph_reader/region_yolo_tests.cpp

```diff
@@ -0,0 +1,150 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <string>
+#include "ngraph_reader_tests.hpp"
+
+TEST_F(NGraphReaderTests, ReadRegionYoloNetwork) {
+    std::string model = R"V0G0N(
+<net name="frozen_graph" version="10">
+	<layers>
+		<layer id="0" name="yolo_out_postprocess/placeholder_port_0" type="Parameter" version="opset1">
+			<data element_type="f32" shape="1,125,13,13"/>
+			<output>
+				<port id="0" precision="FP32">
+					<dim>1</dim>
+					<dim>125</dim>
+					<dim>13</dim>
+					<dim>13</dim>
+				</port>
+			</output>
+		</layer>
+		<layer id="1" name="yolo_out_postprocess/y/Output_0/Data_/copy_const" type="Const" version="opset1">
+			<data offset="0" size="4" shape="1,1,1,1" element_type="f32"/>
+			<output>
+				<port id="1" precision="FP32">
+					<dim>1</dim>
+					<dim>1</dim>
+					<dim>1</dim>
+					<dim>1</dim>
+				</port>
+			</output>
+		</layer>
+		<layer id="2" name="yolo_out_postprocess" type="Multiply" version="opset1" precision="FP32">
```

Comment:
I think we don't need this Multiply in the test for region yolo.

Reply:
yup, done